framework/Dockerfile

```diff
@@ -53,7 +51,8 @@ RUN git clone https://github.com/idrinth-api-bench/framework.git \
     && (rm -r ./test || true) \
     && (rm -r ./tools || true) \
     && (rm -r ./fixtures || true) \
-    && (rm -r ./language || true)
+    && (rm -r ./language || true) \
+    && cp --recursive /api-bench/framework/* /usr/local/lib/node_modules/@idrinth/api-bench/
```

Comment:
I thought you were cloning into /project?

Reply:
replaced /usr/local/lib/node_modules/@idrinth/api-bench/ with /project